woocommerce/sourcebuster-js
# Add more GA4 UTM parameters

### Changes proposed in this Pull Request:
Similar to #1. This PR adds a few more UTM parameters that are recommended / highlighted in [the Google Analytics 4 documentation regarding URL builders](https://support.google.com/analytics/answer/10917952#parameters&zippy=%2Cin-this-article).

The parameters are:
- `utm_source_platform`
- `utm_creative_format`
- `utm_marketing_tactic`

All will default to no value.



### How to test the changes in this Pull Request:
1. Make the built `/public/` directory available to a browser.
2. Copy one of the **UTM** test links at the bottom of the demo page into an (incognito?) browser window.
3. Confirm the three new parameters have values.
4. Click on the other  **UTM** test link at the bottom of the demo page.
5. Confirm that the **first** and **current** values are different.
6. Delete cookies and click the **Demo** link in the nav bar.
7. Confirm that the new parameter values are `(none)`.
8. Manually add the new parameters to the URL: `?utm_source=new-params-test&utm_source_platform=app&utm_creative_format=video&utm_marketing_tactic=onboarding`
9. Confirm they're correctly captured.


### Changelog entry
> Add - GA4 UTM parameters.
